recipes/libntlm/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.4" %}
+
+package:
+  name: libntlm
+  version: "{{ version }}"
+
+source:
+  fn: libntlm-{{ version }}.tar.gz
+  url: http://www.nongnu.org/libntlm/releases/libntlm-{{ version }}.tar.gz
+  md5: 54793488450e9085c57e0d859ff3bc8c
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
same as hdfs3 - please add a comment as to why this is linux-only, and what it might take to get it building elsewhere.